plugins/UserAgreementPlugin/UserAgreement.py

```diff
@@ -0,0 +1,54 @@
+# Copyright (c) 2015 Ultimaker B.V.
```

Comment:
![What year is it?](http://i.lvme.me/q7az4xt.jpg)

(Goes for all of these files.)

Reply:
done